libs/blocks/merch-card/merch-card.js

```diff
@@ -219,6 +219,15 @@ const addStock = (merchCard, styles) => {
   }
 };
 
+const simplifyHrs = (el) => {
+  const hrs = el.querySelectorAll('hr');
+  hrs.forEach((hr) => {
+    if (hr.parentElement.tagName === 'P') {
+      hr.parentElement.replaceWith(hr);
```

Comment:
Is there a use-case where the `hr` has some siblings? Say `<p><hr /> some text</p>`. This logic would leave just a `<hr />`, the text or other nodes would be removed.

Reply:
do you think this is possible with EDS?
after hr, I would expect EDS to render a `p` for the next text node.

Reply:
EDS would likely handle this case, yes. But I see a lot of other parsing going on before this method is called, it might be that some of that decoration adds this type of markup

Reply:
This use case is not possible because for an `hr` with custom color to render it should have its own line where the color or gradient is defined. If the hr (which in authoring starts with `---`) is in the same line with text, the decorator function would not recognize it as a valid hex color or gradient. It would also look very weird in Word for authors to have everything in the same line.
Example: 
<img width="643" alt="Screenshot 2024-01-09 at 9 09 50 AM" src="https://github.com/adobecom/milo/assets/1179549/c33cb086-64f7-4685-a710-f1fba56b0865">